config/300-pubsub.yaml

```diff
@@ -65,6 +65,41 @@ spec:
             sink:
               type: object
               description: "Reference to an object that will resolve to a domain name to use as the sink."
+              anyOf:
+                - type: object
+                  properties:
+                    uri:
+                      type: string
+                      minLength: 1
+                - type: object
+                  properties:
+                    ref:
+                      type: object
+                      required:
+                        - apiVersion
+                        - kind
+                        - name
+                      properties:
+                        apiVersion:
+                          type: string
+                          minLength: 1
+                        kind:
+                          type: string
+                          minLength: 1
+                        name:
+                          type: string
+                          minLength: 1
+                - type: object
```

Comment:
Should this one also have the `required` from above?

Reply:
Probably, but this is the deprecated version. Would you like me to do so?

Reply:
I won't complain either way :smiley: